packages/haskell-regex-tdfa/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE="https://wiki.haskell.org/Regular_expressions"
+TERMUX_PKG_DESCRIPTION="Pure Haskell Tagged DFA Backend for "Text.Regex" (regex-base)"
```

Comment:
Quotes in the description should be escaped or removed

Reply:
Oh! That was a bug in my package creation scripts..